sonyc_kalman/kvae/datasets/box.py

```diff
@@ -0,0 +1,127 @@
+import pygame
```

Comment:
Do we need this file? I think we can exclude it since it seems to be mainly concerned w/ 2D physics